ci-operator/templates/openshift/installer/cluster-launch-installer-kubevirt-e2e.yaml

```diff
@@ -0,0 +1,607 @@
+        export KUBE_SSH_KEY_PATH=/tmp/cluster/ssh-privatekey
+        export TEST_PROVIDER='{"type":"kubevirt"}'
+        mkdir -p ~/.ssh
+        cp /tmp/cluster/ssh-privatekey ~/.ssh/kubevirt_rsa || true
+
+        mkdir -p /tmp/output
+        cd /tmp/output
+
+        function run-upgrade-tests() {
+          openshift-tests run-upgrade "${TEST_SUITE}" --to-image "${IMAGE:-${RELEASE_IMAGE_LATEST}}" \
+            --options "${TEST_OPTIONS:-}" \
+            --provider "${TEST_PROVIDER:-}" -o ${ARTIFACT_DIR}/e2e.log --junit-dir ${ARTIFACT_DIR}/junit
+        }
+
+        function run-tests() {
+          openshift-tests run "${TEST_SUITE}" \
+            --provider "${TEST_PROVIDER:-}" -o ${ARTIFACT_DIR}/e2e.log --junit-dir ${ARTIFACT_DIR}/junit
+        }
+
+        function run-minimal-tests() {
+          # Grab all of the tests marked Feature:Builds and conformance/parallel/minimal
+          openshift-tests run openshift/conformance/parallel --dry-run |
+            grep 'Early' |
+          openshift-tests run -o ${ARTIFACT_DIR}/e2e.log \
+            --junit-dir ${ARTIFACT_DIR}/junit -f -
+        }
+
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1839463
+        # TODO: change the base domain after the infra-cluster would be up and running
+        curl -k --connect-timeout 2 --retry 30 --retry-delay 30 https://api.${OCP_CLUSTER}.gcp.devcluster.openshift.com:6443/apis/config.openshift.io/v1/infrastructures/cluster
+        ${TEST_COMMAND}
+
+    # Runs an install
+    - name: setup
+      # TODO: need to point the IMAGE_KUBEVIRT_INSTALLER to our image in docker.io
+      image: ${IMAGE_KUBEVIRT_INSTALLER}
+      terminationMessagePolicy: FallbackToLogsOnError
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: ARTIFACT_DIR
+        value: /tmp/artifacts
+      - name: TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: kubevirt
+      - name: BASE_DOMAIN
+        value: ${BASE_DOMAIN}
+      - name: SSH_PUB_KEY_PATH
+        value: /etc/openshift-installer/ssh-publickey
+      - name: PULL_SECRET_PATH
+        value: /etc/openshift-installer/pull-secret
+      - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+        value: ${RELEASE_IMAGE_LATEST}
+      - name: OPENSHIFT_INSTALL_INVOKER
+        value: openshift-internal-ci/${JOB_NAME}/${BUILD_ID}
+      - name: USER
+        value: test
+      - name: HOME
+        value: /tmp
+      - name: INSTALL_INITIAL_RELEASE
+        value: ${INSTALL_INITIAL_RELEASE}
+      - name: RELEASE_IMAGE_INITIAL
+        value: ${RELEASE_IMAGE_INITIAL}
+      - name: JOB_NAME
+        value: ${JOB_NAME}
+      command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        # TODO: Wait untill lease is acquired
```

Comment:
comment out all the commands here because we first need to create the lease definition in boskos service. Its less important for now, we can settle with a comment that this is TODO